package.json

```diff
@@ -56,8 +56,9 @@
     "jstz": "~1.0.7",
     "lodash": "~4.15.0",
     "multiselect": "~0.9.12",
+    "prop-types": "^15.5.9",
     "react": "^15.1.0",
```

Comment:
since we have caret (^) npm installs automatically 15.5.4.
anyway i dont think it will conflict since we are using an external package here (prop-types)